src/main/java/arb/logic/commands/project/ListProjectCommand.java

```diff
@@ -32,7 +32,8 @@ public CommandResult execute(Model model, ListType currentListBeingShown) {
         requireNonNull(model);
         model.updateFilteredProjectList(PREDICATE_SHOW_ALL_PROJECTS);
         model.updateSortedProjectList(PROJECT_NO_COMPARATOR);
-        String message = String.format(MESSAGE_PROJECTS_CONTENT, model.getProjectsContent());
+        String formattedMessage = getMessageProjectsContent(model);
+        String message = String.format(MESSAGE_PROJECTS_CONTENT, formattedMessage);
```

Comment:
Doing a `String.format` is unnecessary, as you may notice that `MESSAGE_PROJECTS_CONTENT` is just `%s`, so you're essentially just replacing it with `formattedMessage` in full. You can directly use `formattedMessage` as the `feedbackToUser` of the `CommandResult`

Reply:
Ok noted!